BigQuery/src/BigQueryClient.php

```diff
@@ -856,6 +856,29 @@ public function geography($value)
         return new Geography($value);
     }
 
+    /**
+     * Create a Json object.
+     *
+     * Json represents a value with a data type of
+     * [JSON](https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#json_type)
+     *
+     * Example:
+     * ```
+     * use Google\Cloud\BigQuery\BigQueryClient;
+     *
+     * $bigQuery = new BigQueryClient();
+     * $json = $bigQuery->json('{}');
+     * ```
+     *
+     * @param string|JsonSerializable|int|null $value The JSON string value.
```

Comment:
You'll have to add `JsonSerializable` in the imports for the doc block to reference it appropriately.

Reply:
Or use `\JsonSerializable` here

Reply:
Done